modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -890,6 +930,11 @@ protected function content_template() {
 
 					view.addRenderAttribute( itemTitleKey, {
 						'class': ['e-n-accordion-item-title'],
+						'data-accordion-index': itemCount,
+						'tabindex': defaultState ? ariaExpanded ? 0 : -1 : 0,
```

Comment:
How does this logic work? I am not familiar with this notation. 

I would like it to work exactly as the frontend version.